src/main/java/ChatBro/TaskManager.java

```diff
@@ -0,0 +1,59 @@
+package ChatBro;
+
+import java.util.ArrayList;
+
+public class TaskManager {
```

Comment:
Perhaps you could add JavaDocs and comments for this class and its methods to improve readability?